packages/cli/scripts/v1-v2.js

```diff
@@ -0,0 +1,119 @@
+import { readFileSync, writeFileSync } from 'fs';
+import * as yaml from 'js-yaml';
+
+const args = process.argv.slice(2);
+
+function extractScopes(description) {
+    const scopeRegex = /(?:Required )?scope\(s\): ([\w,:.-]+)/i;
+    const match = scopeRegex.exec(description);
+    return match ? match[1].split(',').map((s) => s.trim()) : '';
+}
+
+function convertYAML(inputYAML) {
+    let data = yaml.load(inputYAML);
+    const location = args[0].replace('nango.yaml', '');
+    let changeLog = [];
+
+    if (data.integrations) {
+        for (const integration in data.integrations) {
+            if (data.integrations[integration]) {
```

Comment:
Is this if necessary. I am not sure how this condition could be not true since `integration in data.integrations`